CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ When releasing a new version:
 
 ### Breaking Changes
 
-_None._
+- Update `AtomicWebServerLogEntry` keys [#711]
```

Comment:
This is added in the wrong place. The "When releasing a new version:" part above has instructions of how to update this file.

Reply:
Since I'm not releasing a new version (yet), I think it's in the right place?

Reply:
This part is a template inside a comment `<!-- ... -->`, [it's not visible in the rendered html](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/2f70f10e34e22abf56fc6ac4d5d6664c9f69d52f/CHANGELOG.md). You'll need to copy this "Unreleased" section and put it outside of the comment block.